libs/designsystem/src/lib/components/page/page.component.ts

```diff
@@ -161,6 +161,7 @@ export class PageComponent
   @Output() enter = new EventEmitter<void>();
   @Output() leave = new EventEmitter<void>();
   @Output() refresh = new EventEmitter<PullToRefreshEvent>();
+  @Output() backButtonClicked = new EventEmitter<Event>();
```

Comment:
Looking at this again i think we should change the name to be in imperative form, considering other events are named like that as well :D 
```suggestion
  @Output() backButtonClick = new EventEmitter<Event>();
```